host/compile.sh

```diff
@@ -60,16 +63,17 @@ if [ $FRAMEWORK = 'cf' ]; then
 		--prototxt ${MODEL_UNZIP}/quantized/deploy.prototxt \
```

Comment:
@yunqu I did not make any changes to these paths, but actually this fails with v1.0 and v1.2 models. I haven't managed to download any v1.1 models so I'm not sure about that.
see:
https://github.com/balintmaci/DPU-PYNQ/commit/9bbbb45c5e9f8aff31ec76641c3ca2f0cc66f578
https://github.com/balintmaci/DPU-PYNQ/commit/e0db5119189d202b585a117a94300f6df20b1c20

Should we include any of these changes?

Reply:
No worries. I will merge your changes in later and make changes on top of it. I think @seujingwei already made some changes on the other PR. We will work this out. Thanks for the contribution.